lca-cli/postpivot/postpivot_test.go

```diff
@@ -876,3 +879,120 @@ func TestProxyAndProxyStatus(t *testing.T) {
 		})
 	}
 }
+
+// test nodeLabelsProvided
+func TestNodeLabelsProvided(t *testing.T) {
+	testcases := []struct {
+		name           string
+		nodeLabels     map[string]string
+		labelsToFilter map[string]string
+		node           *corev1.Node
+		expectedError  bool
+	}{
+		{
+			name: "Happy flow",
+			nodeLabels: map[string]string{
+				"test": "test",
+			},
+			node: &corev1.Node{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-node",
+					Labels: map[string]string{
+						"node-role.kubernetes.io/master": "",
+					},
+				},
+			},
+			expectedError: false,
+		},
+		{
+			name: "Happy flow filter default values test",
+			nodeLabels: map[string]string{
+				"test":                                  "test",
+				"beta.kubernetes.io/arch":               "test",
+				"beta.kubernetes.io/os":                 "test",
+				"kubernetes.io/arch":                    "test",
+				"kubernetes.io/hostname":                "test",
+				"kubernetes.io/os":                      "test",
+				"node-role.kubernetes.io/control-plane": "test",
+				"node.openshift.io/os_id":               "test",
+				"node-role.kubernetes.io/worker":        "test",
+			},
+			labelsToFilter: map[string]string{
+				"beta.kubernetes.io/arch":               "test",
+				"beta.kubernetes.io/os":                 "test",
+				"kubernetes.io/arch":                    "test",
+				"kubernetes.io/hostname":                "test",
+				"kubernetes.io/os":                      "test",
+				"node-role.kubernetes.io/control-plane": "test",
+				"node.openshift.io/os_id":               "test",
+				"node-role.kubernetes.io/worker":        "test",
+			},
+			node: &corev1.Node{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-node",
+					Labels: map[string]string{
+						"node-role.kubernetes.io/master": "",
+					},
+				},
+			},
+			expectedError: false,
+		},
+		{
+			name:          "No labels provided",
+			nodeLabels:    map[string]string{},
```

Comment:
This would be equivalent to using a seed image generated without this update, thanks to the omitempty? Such that a seed image generated with a previous LCA can be handled by this change? The code looks like it handles this case, just wanted to confirm

Reply:
That what i meant in this test case

Reply:
let me think how i can validate it

Reply:
It is more like , post pivot should be able to run even if target doesn't set the field. 
Ill check manually if there is any issue if target sets the field but seed has old version of post-pivot